cmd/hologram-agent/main.go

```diff
@@ -36,6 +36,21 @@ var (
 	config      Config
 )
 
+func ensureCIDR(s string) (*net.IPNet, error) {
+	_, ipNet, err := net.ParseCIDR(s)
+	if err == nil {
+		return ipNet, nil
+	}
+	log.Warning(err.Error())
```

Comment:
This and the other warning in this function are really the responsibility of the caller (which already prints a similar warning, into which this actual error could be incorporated).